backend/tests/test_ingest_statements.py

```diff
@@ -0,0 +1,90 @@
+from composer.services.cs_ingestion.cs_ingestion_services import ingest_statements
+from composer.models import ConnectivityStatement
+import unittest
+from composer.services.cs_ingestion.neurondm_script import log_error
+from composer.enums import CSState
+from django.db.models import Q
+
+
+class TestIngestStatements(unittest.TestCase):
```

Comment:
suggestion (non-blocking): Use django.TestCase instead of unittest.TestCase. As mentioned privately, as a rule of thumb, when the tests require database access I prefer to use django's TestCase instead of unittest.

Reply:
Updated! thanks :)